DESIGN.md

```diff
@@ -0,0 +1,78 @@
+
+## **Design Philosophy and UX Concept**
+
+- **User-Centric Design**: Finesse frontend is developed with a user-first approach. This design focus ensures a simple and intuitive search experience, enhancing user engagement and making the interface effortlessly accessible for a diverse user base.
```

Comment:
Review what is generic to all frontend components in terms of UX and UI and move to a dev-rel-docs PR (and point to it from this DESIGN.md)